configs/rules.json

```diff
@@ -354,6 +354,12 @@
       }
     }
   },
+  "accessibility": {
+    "max_days_in_cache": 7,
+    "days_lookup": 7,
+    "confidence_threshold": 0.9,
```

Comment:
It would be nice to move `confidence_threshold` to be a constructor parameter. See: https://github.com/mozilla/bugbot/issues/1437